src/components/container-with-background.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box, Container } from 'theme-ui';
+
+export interface ContainerWithBackgroundProps {
+  color?: 'green' | 'beige';
+}
+
+export const ContainerWithBackground: React.FC<
```

Comment:
Do we need this? Does it need to be two components?

Reply:
We use this container on different backgrounds so it seems reasonable to have this color check IMO. Might be some other way to do it though!